Makefile

```diff
@@ -102,7 +102,7 @@ debug:
 	@echo PATCH:"$(PATCH)"
 
 # These tags make sure we can statically link and avoid shared dependencies
```

Comment:
This comment is no longer accurate, we should have removed it before when we enabled cgo.

Reply:
we should address this in separate PR